### ISSUE TITLE
misaligned node labels in mermaid diagrams

### ISSUE THREAD

**Issue by samuelpmish** [typora/typora-issues#5812]
When drawing diagrams with mermaid in Typora on my mac, the rendered output looks different than all the examples shown here: https://support.typora.io/Draw-Diagrams-With-Markdown/#mermaid

Specifically, the labels for each node are vertically offset from the center. 
On my machine with v1.6.7
<img width="1007" alt="Screenshot 2023-08-28 at 12 07 56 PM" src="https://github.com/typora/typora-issues/assets/16393433/411f998f-11b6-4ef2-b8cd-95476864ea96">

compared with
(github)
```mermaid
graph LR
a-->B
```

Is this configurable, or a bug in the formatting? The centered labels look much cleaner to me.

-------

Another comparison:
(typora v1.6.7)
<img width="882" alt="Screenshot 2023-08-28 at 12 08 51 PM" src="https://github.com/typora/typora-issues/assets/16393433/3be0d75a-59d4-42cb-b94a-12563eb6c270">

(github)
```mermaid
graph LR
A[Hard edge] -->B(Round edge)
    B --> C{Decision}
    C -->|One| D[Result one]
    C -->|Two| E[Result two]
```

**Comment by samuelpmish** [comment]
update: same issue with v1.7.2-dev with its newer version of `mermaid` on mac

**Comment by samuelpmish** [comment]
v1.7 on a linux version of typora centers the labels correctly

**Comment by abnerlee** [comment]
fixed in 1.7.5

**Comment by samuelpmish** [comment]
the problem appears to not be fixed in 1.7.5 on mac

<img width="1016" alt="Screenshot 2023-09-23 at 8 41 55 AM" src="https://github.com/typora/typora-issues/assets/16393433/58f9243e-11f0-48da-8217-866f80408cbd">
